Makefile

```diff
@@ -3,11 +3,20 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
+TESTTARGETS := github.com/openshift/dedicated-admin-operator/pkg/controller/namespace github.com/openshift/dedicated-admin-operator/pkg/dedicatedadmin
+# ex, -v
+TESTOPTS := 
+
```

Comment:
I haven't seen this syntax in a makefile before. Specifically `:=`. Does this run?

Reply:
The `:=` is called a "simply expanded" variable setting. Check out https://www.oreilly.com/library/view/managing-projects-with/0596006101/ch03.html#variable_types for a really good look at the types.

Reply:
Nice, thanks for that info! `:=` can be hard to google for. ha